plugin/src/main/resources/db/migration/V0.1.0__Init_Version.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS BUILDS (
```

Comment:
Initial migration will be executed on non-empty db schema. 
I think it should be done after initial migration